folio-spring-i18n/pom.xml

```diff
@@ -0,0 +1,64 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>org.folio</groupId>
+    <artifactId>folio-spring-support</artifactId>
+    <version>7.3.0-SNAPSHOT</version>
+  </parent>
+
+  <name>folio-spring-i18n</name>
+  <artifactId>folio-spring-i18n</artifactId>
+  <description />
```

Comment:
The tiniest of nits here: do we really need this `description` tag?

Reply:
Idk...I copied this from folio-spring-system-user as a starting point lol. I'll write an actual description